boefjes/boefjes/plugins/kat_nmap/normalize.py

```diff
@@ -1,5 +1,5 @@
 import logging
-from typing import Iterator, Union
```

Comment:
Iterator is still used on line 20 in the signature of get_ip_ports_and_service()